include/glow/Backends/BackendUtils.h

```diff
@@ -21,6 +21,43 @@
 #include "glow/IR/IR.h"
 
 namespace glow {
+namespace runtime {
+/// RuntimeSymbolInfo
+/// Contains information for initialization and handling of symbol at runtime.
+struct RuntimeSymbolInfo {
+  /// The size in bytes.
+  size_t size;
+  /// Offset in bytes from the base address.
+  size_t offset;
+  /// Type of symbol.
+  Type type;
+};
+/// Runtime Bundle
+/// Contains the information needed to be passed forward from compile time to
+/// runtime. In order to allocate and initialize memory.
+class RuntimeBundle {
+public:
```

Comment:
Typically, all members of structs are public by default and all members of classes are private by default. And if some internals of the class need to be exposed, it is done via getters and setter. Also the private members would have the "_" suffix in their names.

So, why is `RuntimeBundle` a class, if all its members are public?